### PR TITLE
Fix linting

### PR DESCRIPTION
**Pull request by pmprete**
fix issues showed in `npm run eslint`

### CODE REVIEW

File: scripts/commissionSplitter/changerHelper.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
+/* eslint-disable no-console */
 const Web3 = require('web3');
 const HDWalletProvider = require('truffle-hdwallet-provider');
 const abiDecoder = require('abi-decoder');
 const _ = require('lodash');
-const jsonfile = require('jsonfile');
```
Comment (shanshi) [inline scripts/commissionSplitter/changerHelper.js:5]: Why did you remove this line?

File: scripts/commissionSplitter/deployCommissionSplitter.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
-/* eslint-disable import/no-unresolved */
-const CommissionSplitter = require('../../build/contracts/CommissionSplitter.json');
-const { deployContract, getConfig, deployProxyContract } = require('./changerHelper');
+/* eslint-disable no-console */
+const { deployProxyContract } = require('./changerHelper');
```
Comment (shanshi) [inline scripts/commissionSplitter/deployCommissionSplitter.js:2]: Why did you change this line?

**Review by shanshi — CHANGES_REQUESTED** [review]

File: scripts/commissionSplitter/changerHelper.js (hunk 1 above)
Comment (pmprete) [reply scripts/commissionSplitter/changerHelper.js:5]: jsonfile was not being used

File: scripts/commissionSplitter/deployCommissionSplitter.js (hunk 2 above)
Comment (pmprete) [reply scripts/commissionSplitter/deployCommissionSplitter.js:2]: deployContract and getConfig where not being used so the linter was complaining about them being imported

**Review by shanshi — APPROVED** [review]
